<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveAsDialog</class>
 <widget class="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save As</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWidget" name="outputCelFileEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="outputCelFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Overwrite all:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="autoOverwriteCheckBox">
        <property name="toolTip">
         <string>Overwrite files without confirmation-request</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="celSettingsGroupBox">
     <property name="title">
      <string>CEL/CL2 File Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Clipped:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="celClippedYesRadioButton">
          <property name="text">
           <string>YES</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">celClippedButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="celClippedNoRadioButton">
          <property name="text">
           <string>NO</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">celClippedButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="celClippedAutoRadioButton">
          <property name="text">
           <string>Preserve</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">celClippedButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Groups:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWidget" name="celGroupEdit">
        <property name="charWidth">
         <number>5</number>
        </property>
        <property name="toolTip">
         <string>Number of groups</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="celClippedButtonGroup"/>
 </buttongroups>
</ui>
